priv/repo/migrations/20170308222552_create_auth_token.exs

```diff
@@ -0,0 +1,14 @@
+defmodule CodeCorps.Repo.Migrations.CreateAuthToken do
+  use Ecto.Migration
+
+  def change do
+    create table(:auth_token) do
+      add :value, :string
+      add :user_id, references(:users, on_delete: :nothing)
```

Comment:
If I'm reading the documentation [here](https://hexdocs.pm/ecto/Ecto.Migration.html#references/2) correctly, this is basically us telling ecto, "if the referenced user is deleted, leave their tokens and do nothing". Basically, it's the other way around from what you're thinking it is. I would change this to `on_delete: :delete_all`.

It's a bit confusing, but this [stackoverflow answer](http://stackoverflow.com/a/34692705) explains it.

In a migration, `references(:parent, on_delete: :action)` will perform the specified action on associated records in this table, when a record in `:parent` gets deleted.

In a schema, `has_many :children, on_delete: :action` will perform the action on the children when the record belonging to this schema is deleted.

But to put it short, just changing to `on_delete: :delete_all` will do the trick here.